### ISSUE TITLE
set resolveDir for asset paths?

### ISSUE THREAD

**Issue by aredridel** [apeschar/esbuild-vue#18]
Hi! 👋 
     
I ran into a problem today where with `esbuild-vue` and `extractCss: true`. I have paths to assets in the CSS, and without the resolveDir, this yields errors similar to this:

```
The plugin "vue" didn't set a resolve directory for the file "esbuild-vue-css:public/src/components/X.vue?esbuild-vue-css", so esbuild did not search for "../scss/backgrounds/y.svg" on the file system.
```

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/esbuild-vue/src/index.js b/node_modules/esbuild-vue/src/index.js
index 7412d22..b86e0c1 100644
--- a/node_modules/esbuild-vue/src/index.js
+++ b/node_modules/esbuild-vue/src/index.js
@@ -1,4 +1,4 @@
-const { resolve, relative } = require("path");
+const { resolve, relative, dirname } = require("path");
 const fs = require("fs");
 const escapeRegExp = require("lodash.escaperegexp");
 
@@ -83,7 +83,7 @@ module.exports = function ({
             if (!css) {
               return null;
             }
-            return { contents: css, loader: "css" };
+            return { contents: css, loader: "css", resolveDir: dirname(path) };
           }
         );
       }
```

**Comment by apeschar** [comment]
Thanks for debugging this. I've included your fix in release 1.2.2. After updating it should work.

**Comment by aredridel** [comment]
Thank you!
